docker/php/Dockerfile

```diff
@@ -39,15 +45,15 @@ RUN extBuildDeps=" \
             libzip-dev \
             sendmail && \
     rm -rf /var/lib/apt/lists/* && \
-    docker-php-ext-configure gd --with-jpeg-dir=/usr/lib/x86_64-linux-gnu --with-png-dir=/usr/lib/x86_64-linux-gnu --with-freetype-dir=/usr/lib/x86_64-linux-gnu && \
+    docker-php-ext-configure gd --enable-gd --with-webp --with-avif --with-jpeg --with-freetype && \
```

Comment:
no `--with-png`?

Reply:
The png option has been dropped by extension.